smoke-tests-otel-starter/spring-boot-common/src/main/java/io/opentelemetry/spring/smoketest/AbstractOtelSpringStarterSmokeTest.java

```diff
@@ -157,7 +164,12 @@ void shouldSendTelemetry() {
                                             AbstractCharSequenceAssert::isNotBlank)))
                         .hasAttribute(HttpAttributes.HTTP_REQUEST_METHOD, "GET")
                         .hasAttribute(HttpAttributes.HTTP_RESPONSE_STATUS_CODE, 200L)
-                        .hasAttribute(HttpAttributes.HTTP_ROUTE, "/ping")));
+                        .hasAttribute(HttpAttributes.HTTP_ROUTE, "/ping")
+                        .hasAttribute(ServerAttributes.SERVER_ADDRESS, "localhost")
+                        .hasAttribute(ClientAttributes.CLIENT_ADDRESS, "127.0.0.1")
+                        .hasAttribute(
+                            OpenTelemetryAssertions.satisfies(
+                                ServerAttributes.SERVER_PORT, AbstractLongAssert::isNotZero))));
```

Comment:
instead of having multiple `hasAttribute` checks we usually add a single `hasAttributesSatisfying` or `hasAttributesSatisfyingExactly`